v_4_0_0/filemanager.go

```diff
@@ -5,11 +5,14 @@ import (
 	"encoding/base64"
 	"os"
 	"path/filepath"
+	"runtime"
 	"text/template"
 
 	"github.com/giantswarm/microerror"
 )
 
+const FilesDir = "v_4_0_0/files"
```

Comment:
I'd rather call it version and set the value to `v_4_0_0`, then `GetIgnitionPath` becomes clearer. Also it doesn't need to be exported.